presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationPushDown.java

```diff
@@ -108,8 +108,7 @@ public void stopKafka()
     public void testPartitionPushDown() throws ExecutionException, InterruptedException
     {
         createMessages(topicNamePartition);
-        String sql = format("SELECT count(*) FROM default.%s WHERE _partition_id=1",
```

Comment:
Is it possible to enforce this kind of formatting somewhere? (airlift?)
No change request.

Reply:
I am not aware of how it could be done.